my-diploma/src/pages/PokemonDetail/PokemonDetalContainers/PokemonDetailContainer.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { useDispatch } from "react-redux";
+import { useSelector } from "react-redux";
```

Comment:
refactor imports. 
1. useDispatch and useSelector must be imported from single import statement
2. Add space lines between local imports and imports from libs. Fix in all app